handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -46,10 +55,18 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 ## Interview process
 
 1. You [apply here](https://jobs.lever.co/sourcegraph/b2f9a8b0-cc06-4629-81a0-0f2fa64271c7/apply).
+1. We screen your application for suitability for the role by looking at:
+   1. Your past work experience. In the absence of professional work experience we would look at any open-source projects you have created or contribute to.
+   1. Whether you have the necessary technical skills that we require.
+   1. The answers you provide to the application questions.
+   1. And the [Nice-to-haves](#nice-to-haves) we listed above.
 1. We set up a 30-minute call to learn more about what you are looking for, tell you about Sourcegraph, and answer any questions that you have.
-1. You complete a 3-hour [exercise project](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test the technical qualifications listed above.
-1. We review your submission and schedule a **1 hour follow-up call** to discuss it.
-1. We schedule additional interviews across multiple days.
+1. You complete a 3-hour [coding excercise](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test the technical qualifications and responsibilities of the role as [listed above](#your-responsibilities).
+   1. _Note: while we do not disqualify candidates who do not have React and TypeScript experience from taking the excercise, you will be required to use these for the excercise._
```

Comment:
@felixfbecker this is the change to the coding exercise that makes it explicit that you should use our tech stack to complete the coding excercise.

Reply:
The thought is that while we don't require you to have professional React and TypeScript experience, the reality is that we exclusively use that in our frontend and we want to assess candidates on their ability to utilize these to create a frontend solution. It places an expectation on the candidate that to pass our coding exercise you need to show us that you can make use of these technologies to deliver your solution. This also ensures we can calibrate our reviews of candidates better since we are not comparing different tech stacks.